_posts/04-02-01-Composer-and-Packagist.md

```diff
@@ -3,138 +3,106 @@ isChild: true
 anchor:  composer_and_packagist
 ---
 
-## Composer and Packagist {#composer_and_packagist_title}
+## Composer a Packagist {#composer_and_packagist_title}
 
-Composer is a **brilliant** dependency manager for PHP. List your project's dependencies in a `composer.json` file and,
-with a few simple commands, Composer will automatically download your project's dependencies and setup autoloading for
-you. Composer is analogous to NPM in the node.js world, or Bundler in the Ruby world.
+Composer je **geniálny** správca závislostí pre PHP. Vymenúva závislosti vášho projektu v súbore `composer.json` a s niekoľkými jednoduchými príkazmi za vás dokáže stiahnuť projektové závislosti a nastaviť autoloading. Composer je analogický ku NPM vo svete node.js alebo analogický ku Bundler vo svere Ruby.
 
-There are already a lot of PHP libraries that are compatible with Composer, ready to be used in your project. These
-"packages" are listed on [Packagist], the official repository for Composer-compatible PHP libraries.
+Dnes existuje už mnoho PHP knižníc, ktoré sú kompatibilné s Composer a teda sú pripravené pre použitie vo vašom projekte. Tieto "balíčky" sú uvedené v oficiálnom repozitári [Packagist] pre Composer-kompatibilné PHP knižnice.
 
-### How to Install Composer
+### Ako nainštalovať Composer
 
-You can install Composer locally (in your current working directory) or globally (e.g. /usr/local/bin, recommended).
-Let's assume you want to install Composer globally:
+Composer je možné nainštalovať lokálne (vo vašom aktuálnom pracovnom adresári) alebo globálne (napr. /usr/local/bin - odporúčané).
+Predpokladajme, že chcete nainštalovať Composer globálne:
 
 {% highlight console %}
 curl -sS https://getcomposer.org/installer | php
 mv composer.phar /usr/local/bin/composer
 {% endhighlight %}
 
-**Note:** If the above fails due to permissions, run the `mv` line again with `sudo`.
+**Poznámka:** Ak vyššie uvedené riadky zlyhajú kvôli právam, opätovne spustite riadok `mv` začínajúci so `sudo`.
 
-This will download `composer.phar` (a PHP binary archive). You can run this with `php` to manage your project
-dependencies.
+Vyššie uvedené riadky stiahnu `composer.phar` (PHP binárny archív). Pre správu vašich závislostí ho môžete spúšťať s `php`.
 
-**Please Note:** If you pipe downloaded code directly into an interpreter, please read the
-code online first to confirm it is safe.
+**Bezpečnostná poznámka:** Ak chcete vložiť uvedený kód priamo do interpretera, najprv si ho tu prečítajte aby ste sa presvedčili, že je bezpečný.
 
-#### Installing on Windows
+#### Inštalácia na Windows
```

Comment:
`Inštalácia na Windows` by malo byť tiež vyskloňované:
* Inštalácia na Windowse
* Inštalácia na systéme Windows